pkg/order/request.go

```diff
@@ -99,11 +176,13 @@ type AddressRequest struct {
 	State        string `json:"state,omitempty"`
 	City         string `json:"city,omitempty"`
 	Complement   string `json:"complement,omitempty"`
-	Floor        string `json:"floor,omitempty"`
 }
 
 type ItemsRequest struct {
 	Title        string `json:"title,omitempty"`
+	Type         string `json:"type,omitempty"`
+	Warranty     bool   `json:"warranty,omitempty"`
+	Event_date   string `json:"event_date,omitempty"`
```

Comment:
[req] Seguir padrão PascalCase na nomenclatura
```suggestion
	EventDate   string `json:"event_date,omitempty"`
```

Reply:
Foi ajustado para padrão PascalCase .